packages/itwinui-css/src/slider/slider.scss

```diff
@@ -84,6 +84,7 @@ $iui-slider-tick-thickness: 1px;
       'min';
     justify-items: center;
     inline-size: fit-content;
+    block-size: 100%;
```

Comment:
unrelated to react (undo or move to #1469 if necessary)

Reply:
Linking the relevant discussion: https://github.com/iTwin/iTwinUI/pull/1469#discussion_r1341785009